macos/package_files/5.0.0/preinstall.sh

```diff
@@ -143,10 +143,8 @@ chown root:wheel /Library/StartupItems/WAZUH
 sudo tee /Library/StartupItems/WAZUH/WAZUH <<-'EOF'
 #!/bin/sh
 . /etc/rc.common
-. /etc/ossec-init.conf
-if [ "X${DIRECTORY}" = "X" ]; then
-    DIRECTORY="/Library/Ossec"
-fi
```

Comment:
Why have you deleted this?

Reply:
It has been deleted because DIRECTORY is loaded from ossec-init.conf, which is not available anymore.
On the other hand, the macOS installation path is mandatorily `/Library/Ossec`, so, the check isn't required and can be solved directly by using the unique valid installation path.